### PR TITLE
Adding test cases.  Adding support for specifying port on route.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

**Comment by sdodson** [comment]
I defer to Tim and Andrew.

File: roles/lib_openshift/src/test/unit/oc_route.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+            'cert_content': None,
+            'key_content': None,
+            'service_name': 'testservice',
+            'host': 'test.openshift.com',
+            'wildcard_policy': None,
+            'weight': None,
+            'port': None
+        }
+
+        route_result ='''{
```
Comment (abutcher) [inline roles/lib_openshift/src/test/unit/oc_route.py]: pyflake: need a space after this `=`

File: roles/lib_openshift/src/test/unit/oc_route.py [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+                                "lastTransitionTime": "2017-02-07T01:59:48Z"
+                            }
+                        ],
+                        "wildcardPolicy": "None"
+                    }
+                ]
+            }
+        }'''
+
+
```
Comment (abutcher) [inline roles/lib_openshift/src/test/unit/oc_route.py]: pyflake: extra blank line here

**Comment by tbielawa** [comment]
> Few flakes from travis otherwise lgtm

Same.

**Comment by kwoodson** [comment]
@tbielawa, @abutcher, Fixed the flake8 issues.  I added a rebase to ensure everything is up-to-date.

**Comment by abutcher** [comment]
aos-ci-test

**Pull request merged**